apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -6040,6 +6046,26 @@ cors_returned_on_get_request(_Config) ->
     "example.com" = proplists:get_value("access-control-allow-origin", Headers),
     ok.
 
+%% ============================================================
+%% Test HTTP cache headers
+%% ============================================================
+
+cache_headers(_Config) ->
+    Host = external_address(),
+    {ok, {{_, 200, _}, Headers, _Body}} =
+        httpc_request(get, {Host ++ "/v2/blocks/top", []}, [], []),
+
+    true = proplists:is_defined("expires", Headers),
+    true = proplists:is_defined("etag", Headers),
```

Comment:
I am not seeing where the actual usage of etag is tested. I expect a conditional get request (is [If-None-Match](https://tools.ietf.org/html/rfc7232#section-3.2) the tag sent by the client?), checking that the response has correct status code (not 200) and that response has not the full blown response body.

Reply:
An additional test case, testing this, would be beneficial. So to check that the functionality actually works.

(BTW It may be done at several layers - either system or integration test.)

Reply:
Etag integration test added in d477b34c693b25d9ae3496333ff37b1133e198a8